ui/design/src/components/Avatar/index.tsx

```diff
@@ -4,7 +4,7 @@ import CommonInterface from '../../interfaces/common.interface';
 import MarginInterface from '../../interfaces/margin.interface';
 import AvatarImage from './avatar-image';
 // import { loadPlaceholder } from './placeholders';
-import StyledAvatar, { AvatarSize, ActiveOverlay } from './styled-avatar';
+import StyledAvatar, { AvatarSize, ActiveOverlay, AvatarBorderColor } from './styled-avatar';
```

Comment:
did you mean AvatarType? because `dev` or `publisher` is not a color

Reply:
I defined those in the grommet theme

Reply:
you didn't defined neither one in grommet theme.
there is no color named `dev` nor `publisher`